storybook/src/stories/UseResizeObserver/ContainerQuery.tsx

```diff
@@ -3,15 +3,15 @@ import { useResizeObserver } from 'too-many-hooks'
 
 const ContainerQuery: React.FC = () => {
   const [container, setContainer] = useState<HTMLDivElement | null>(null)
-  const entry = useResizeObserver(container)
+  const entries = useResizeObserver(container)
 
   return (
     <div className="prose flex flex-col items-center gap-4">
       <p>Resize to change the flow of content</p>
       <div
         ref={element => setContainer(element)}
         className={`flex h-80 w-80 resize gap-4 overflow-auto border border-slate-800 bg-slate-200 p-4 ${
-          entry && entry.contentRect.width < 300 ? 'flex-col' : ''
+          entries && entries[0].contentRect.width < 300 ? 'flex-col' : ''
```

Comment:
Is it ever possible for `entries` to be an empty array? If so, we'll need to check the length when we access `entries[0]`

```suggestion
          entries?.length && entries[0].contentRect.width < 300 ? 'flex-col' : ''
```

Reply:
I actually think with the current implementation it cannot occur. By default, `entries` is `undefined` and it does not change until a resize is observed, at which point it is set to the full array of entries. I could change it to default to an empty array if we think that is better but I like the default being `undefined` so you can just check if it exists rather than if it has length,